### PR TITLE
Add upack changes back into the task. Fix help markdowns for inputs.

### PR DESCRIPTION
**Pull request by aasim**

### CODE REVIEW

File: Tasks/DownloadPackageV1/task.json [hunk 1]
```diff
@@ -69,7 +68,7 @@
             "properties": {
                 "EditableOptions": "True"
             },
-            "helpMarkDown": "Select the package to download."
+            "helpMarkDown": "If you don't find the package in the list, you can provide the package ID, which you can find using the instructions [here](https://docs.microsoft.com/en-us/rest/api/azure/devops/?view=azure-devops-rest-5.0#get-packages)."
```
Comment (shubham90) [inline Tasks/DownloadPackageV1/task.json]: Can we have a fwdlink instead of hard coded url?

File: Tasks/DownloadPackageV1/task.json [hunk 2]
```diff
@@ -93,7 +92,7 @@
                 "EditableOptions": "True"
             },
             "groupName": "advancedOptions",
-            "helpMarkDown": "Set the files pattern filter"
+            "helpMarkDown": "Specify which files to download using [file matching patterns](https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/file-matching-patterns?view=azure-devops)."
```
Comment (shubham90) [inline Tasks/DownloadPackageV1/task.json]: same comment about fwdlink

File: Tasks/DownloadPackageV1/task.json (hunk 1 above)
Comment (aasim) [reply Tasks/DownloadPackageV1/task.json]: Fixed

File: Tasks/DownloadPackageV1/task.json (hunk 2 above)
Comment (aasim) [reply Tasks/DownloadPackageV1/task.json]: Fixed

**Pull request merged**